docs/source/developer/contributing.rst

```diff
@@ -431,6 +431,16 @@ includes a ``Vagrantfile``, the bootstrap files and additional documentation.
 Installing JupyterLab
 ---------------------
 
+Important for Windows users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important::
+
+   On Windows, symbolic links need to be activated on Windows 10 or above for Python version 3.8 or higher
```

Comment:
Thanks for adding this section! Since it duplicates the existing section in extension tutorial:

https://github.com/jupyterlab/jupyterlab/blob/9746d1599ec2056f9a3c3f928214623a00e4e550/docs/source/extension/extension_tutorial.rst?plain=1#L206

I am slightly concerned that with time someone will edit this or that notice and they will diverge. Can you add a comment here and there to remind anyone who might be editing this section to also update the other instance?

See https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments for comment syntax in RST